Snowflake.Data.Tests/IntegrationTests/StructuredTypesIT.cs

```diff
@@ -0,0 +1,1471 @@
+                        'DateTimeOffsetValue', NULL,
+                        'TimeSpanValue', NULL,
+                        'BinaryValue', NULL,
+                        'SemiStructuredValue', NULL
+                    )::OBJECT(
+                        StringValue VARCHAR,
+                        CharValue CHAR,
+                        ByteValue SMALLINT,
+                        SByteValue SMALLINT,
+                        ShortValue SMALLINT,
+                        UShortValue INTEGER,
+                        IntValue INTEGER,
+                        UIntValue INTEGER,
+                        LongValue BIGINT,
+                        ULongValue BIGINT,
+                        FloatValue FLOAT,
+                        DoubleValue DOUBLE,
+                        DecimalValue REAL,
+                        BooleanValue BOOLEAN,
+                        GuidValue TEXT,
+                        DateTimeValue TIMESTAMP_NTZ,
+                        DateTimeOffsetValue TIMESTAMP_LTZ,
+                        TimeSpanValue TIME,
+                        BinaryValue BINARY,
+                        SemiStructuredValue OBJECT
+                    )";
+                    command.CommandText = $"SELECT {allTypesObjectAsSFString}";
+                    var reader = (SnowflakeDbDataReader)command.ExecuteReader();
+                    Assert.IsTrue(reader.Read());
+
+                    // act
+                    var allTypesObject = reader.GetObject<AllNullableTypesClass>(0);
+
+                    // assert
+                    Assert.NotNull(allTypesObject);
+                    Assert.IsNull(allTypesObject.StringValue);
+                    Assert.IsNull(allTypesObject.CharValue);
+                    Assert.IsNull(allTypesObject.ByteValue);
+                    Assert.IsNull(allTypesObject.SByteValue);
+                    Assert.IsNull(allTypesObject.ShortValue);
+                    Assert.IsNull(allTypesObject.UShortValue);
+                    Assert.IsNull(allTypesObject.IntValue);
+                    Assert.IsNull(allTypesObject.UIntValue);
+                    Assert.IsNull(allTypesObject.LongValue);
+                    Assert.IsNull(allTypesObject.ULongValue);
+                    Assert.IsNull(allTypesObject.FloatValue);
+                    Assert.IsNull(allTypesObject.DoubleValue);
+                    Assert.IsNull(allTypesObject.DecimalValue);
+                    Assert.IsNull(allTypesObject.BooleanValue);
+                    Assert.IsNull(allTypesObject.GuidValue);
+                    Assert.IsNull(allTypesObject.DateTimeValue);
+                    Assert.IsNull(allTypesObject.DateTimeOffsetValue);
+                    Assert.IsNull(allTypesObject.TimeSpanValue);
+                    Assert.IsNull(allTypesObject.BinaryValue);
+                    Assert.IsNull(allTypesObject.SemiStructuredValue);
+                }
+            }
+        }
+
+        [Test]
+        public void TestSelectMapSkippingNullValues()
```

Comment:
Do we need that test? It tests backend feature, not driver.

Reply:
I removed it